### PR TITLE
Add all post call to get effective routes

### PR DESCRIPTION
**Pull request by zaynchen115**
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Network/Network/Az.Network.psd1 [hunk 1]
```diff
@@ -436,6 +436,8 @@ CmdletsToExport = 'Add-AzApplicationGatewayAuthenticationCertificate',
                'Add-AzVirtualHubRoute', 'New-AzVirtualHubRouteTable', 
                'Add-AzVirtualHubRouteTable', 'Get-AzVirtualHubRouteTable', 
                'Remove-AzVirtualHubRouteTable', 'New-AzRoutingPolicy', 
+               'Get-AzVHubEffectiveRoute', 'Get-AzVHubInboundRoute',
```
Comment (msJinLei) [inline src/Network/Network/Az.Network.psd1:439]: @zaynchen115 
Refer to our [naming best practice](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#specific-noun-and-noun-singularity), you should use singular name for Get-AzVHubEffectiveRoutes, Get-AzVHubInboundRoutes and Get-AzVHubOutboundRoutes. As you make breaking change to a new added cmdlet `GetAzureRmVirtualHubEffectiveRoutesCommand`, if we don't include your PR into this release, the new added cmdlet will be not allowed to change in a breaking way until next major release. And so we change your cmdlet names from plural to singular and include them into this release.
Please follow the code-freeze date for the next time.

**Comment by msJinLei** [comment]
Use this PR to replace https://github.com/Azure/azure-powershell/pull/21745

@zaynchen115 
Refer to our [naming best practice](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#specific-noun-and-noun-singularity), you should use singular name for Get-AzVHubEffectiveRoutes, Get-AzVHubInboundRoutes and Get-AzVHubOutboundRoutes. As you make breaking change to a new added cmdlet `GetAzureRmVirtualHubEffectiveRoutesCommand`, if we don't include your PR into this release, the new added cmdlet will be not allowed to change in a breaking way until next major release. And so we change your cmdlet names from plural to singular and include them into this release.

Please follow the code-freeze date for the next time.

File: src/Network/Network/ChangeLog.md [hunk 2]
```diff
@@ -48,6 +48,12 @@
 * Onboarded `Microsoft.HardwareSecurityModules/cloudHsmClusters` to private link cmdlets
 * Updated cmdlet to add the property of `DisableRequestBodyEnforcement`, `RequestBodyInspectLimitInKB` and `DisableFileUploadEnforcement` 
     - `New-AzApplicationGatewayFirewallPolicySetting`
+* Added optional property 'AuxiliarySku' to cmdlet `New-AzNetworkInterface` to help choose performance on an AuxiliaryMode enabled Network Interface.
+* Added a new value, 'AcceleratedConnections' for existing property 'AuxiliaryMode' for `New-AzNetworkInterface`
```
Comment (isra-fel) [inline src/Network/Network/ChangeLog.md:52]: Duplicated changelog with 57-58, probably a bad merge
```suggestion
```

**Review by isra-fel — CHANGES_REQUESTED** [review]
`Az.Network.md` is missing these new cmdlets

File: src/Network/Network/ChangeLog.md (hunk 2 above)
Comment (msJinLei) [reply src/Network/Network/ChangeLog.md]: Resolved

**Comment by msJinLei** [comment]
> `Az.Network.md` is missing these new cmdlets

Update the help folder

**Review by isra-fel — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]

**Pull request merged**